arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,699 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure ML Commitment Plans Management Client",
+    "description": "These APIs allow end users to operate on Azure Machine Learning Commitment Plans resources and their child Commitment Association resources. They support the following operations:<ul><li>Create or update a commitment plan</li><li>Get a commitment plan</li><li>Patch a commitment plan</li><li>Delete a commitment plan</li><li>Get all commitment plans in a resource group</li><li>Get all commitment plans in a subscription</li><li>Get a commitment association</li><li>Get all commitment associations for a commitment plan</li><li>Move a commitment association from one commitment plan to another</li><li>Get usage history for a commitment plan</li></ul>",
+    "version": "2016-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans/{commitmentPlanName}/commitmentAssociations/{commitmentAssociationName}": {
+      "get": {
+        "tags": [
+          "CommitmentAssociations"
+        ],
+        "operationId": "CommitmentAssociations_Get",
+        "description": "Gets a commitment association.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CommitmentPlanNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CommitmentAssociationNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommitmentAssociation"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/commitmentPlans/{commitmentPlanName}/commitmentAssociations": {
+      "get": {
+        "tags": [
+          "CommitmentAssociations"
+        ],
+        "operationId": "CommitmentAssociations_List",
+        "description": "Get all commitment associations for a parent commitment plan.",
```

Comment:
Please apply the extension "x-ms-pageable" on all the pageable operations.
You can find out more info about our extensions over [here](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md). Please read them carefully and apply them wherever applicable.